tests/data_context/store/test_store_backends.py

```diff
@@ -666,89 +666,6 @@ def test_TupleS3StoreBackend_with_prefix(aws_credentials):
     )
 
 
-# NOTE: Chetan - 20211118: I am commenting out this test as it was introduced in: https://github.com/great-expectations/great_expectations/pull/3377
-# We've decided to roll back these changes for the time being since they introduce some unintendend consequences.
```

Comment:
Can you skip it instead of commenting it out?
You can include the note about this in the `skip(reason=...)` and then it will show up in the test output.

Reply:
so this has been commented out for 2 years - I think we should just delete it tbh